terraform/main.tf

```diff
@@ -315,7 +321,9 @@ locals {
       ingestor                                = pair[1]
       kubernetes_namespace                    = kubernetes_namespace.namespaces[pair[0]].metadata[0].name
       packet_decryption_key_kubernetes_secret = kubernetes_secret.ingestion_packet_decryption_keys[pair[0]].metadata[0].name
-      ingestor_manifest_base_url              = var.ingestors[pair[1]]
+      ingestor_manifest_base_url              = var.ingestors[pair[1]].manifest_base_url
+      intake_batch_replica_count              = var.ingestors[pair[1]].localities[pair[0]].intake_batch_replica_count
```

Comment:
minor nit on naming. Are those the replica counts for the intake batch and aggregate workers? If yes, `intake_worker_count` and `aggregate_worker_count` might be more descriptive.